### ISSUE TITLE
A11Y - Table cell missing context - Languages Supported on Search.gov Results Pages

### ISSUE THREAD

**Issue by davidstenger** [GSA/search-gov-website#307]
**Languages Supported on Search.gov Results Pages | Search.gov
Page URL: https://search.gov/admin-center/dashboard/supported-languages.html**

Data tables include header and data cells arranged in a grid.

Each data cell `<td>` should be assigned to at least one column or row header cell `<th>`. When this is done, a screen reader can announce each data cell in context.

**Why is this an issue?**
Sighted users can make a visual association between table header and data cells. This context gives meaning to the data.

Screen reader users must rely on a programmatic relationship being made between header and data cells. As each data cell is read out, the relevant row or column header is also announced. In this way, the data has context.

It is best practice to limit table use to simple data sets. Complex tables are difficult to understand and there may be other, more accessible ways to present the content.

WCAG 2.2: [Understanding Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships)
